ibr/html/index.html

```diff
@@ -9,6 +9,8 @@
 		<script src="../node_modules/three/build/three.js"></script>
 		<script src="../node_modules/three/examples/js/utils/BufferGeometryUtils.js"></script>
 		<script type="module" src="../node_modules/ibr-sdk/build/ibr-sdk.js"></script>
+		<script type="module" src="../node_modules/ibr-sdk/class/Layer.js"></script>
+		<script type="module" src="../node_modules/ibr-sdk/class/IBRObject.js"></script>
```

Comment:
do we have to script tag all these dependencies? Is there a way to have ibr-sdk package all of this for me so i only need to import the ibr-sdk package and not manually manage these myself?

Reply:
done